galasa-simplatform-application/src/main/java/dev/galasa/simplatform/listener/ListenerManager.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 
+ * (c) Copyright IBM Corp. 2019.
+ */
+package dev.galasa.simplatform.listener;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/*
+ * 	ListenerManager acts as a head listener for all current
+ * 	and future listeners. It takes the request from the sender
+ * 	and refers to the correct listener.
+ * 
+ * 	This solves the issue of having multiple listeners 
+ * 	waiting for requests across multiple ports.
+ * 
+ *	Written by @Lewis James on 17/12/2019
+ * 
+ */
+
+public class ListenerManager implements IListener{
+	
+	private Socket socket;
+	private List<String> headers = new ArrayList<>();
+	private String payload = new String();
+	
+    private Logger log = Logger.getLogger("Simplatform");
+
+	public void run() {
+		try {
+			
+			/* Attempt to read the request listened by the server */
+			processInput();
+			
+			/* Interpret which sort of sub-listener we will need to call upon */
+            String path = findPath().trim();
+            
+            /* Switching through the different path options */
+            switch(path) {
+            
+            case "/updateAccount":   	
+            	/* Initialise the CreditAccountListener and send payload */
+            	new CreditAccountListener().sendRequest(this, payload);
+            	break;
+            	
+            case "/processTransfer":
+            	/* Initialise the AccountTransferListener and send payload */
+            	new AccountTransferListener().sendRequest(this, payload);
+            	break;
```

Comment:
would be great if the parent can ask the child for the path that it can process, this means that new listeners could be added without having to update the parent each time